frontend/src/components/layouts/SettingsLayout.tsx

```diff
@@ -117,7 +119,13 @@ export default function SettingsLayout() {
             <MenuItem to="/settings/developer">Developer</MenuItem>
             <MenuItem to="/settings/debug-tools">Debug Tools</MenuItem>
           </nav>
+          {info?.version && (
```

Comment:
Why do we show the version in two places now?

Reply:
![image](https://github.com/user-attachments/assets/2898490d-163a-4e75-b6f6-f2ec3a3606a3)

it also hovers in mid-air on some pages